Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -96,6 +96,7 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
     private fun FragmentRecoveryCertificateDetailsBinding.onCertificateReady(
         certificate: RecoveryCertificate
     ) {
+        toolbar.menu.findItem(R.id.menu_recovery_certificate_export).isVisible = certificate.isNotBlocked
```

Comment:
I guess unavailable actions should be displayed as disabled rather than removing them. At least according to the material design guidelines.
https://material.io/components/menus#anatomy

Reply:
done